packages/ERTP/jsconfig.json

```diff
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
```

Comment:
What does this mean?

Reply:
It means that if there are standards-track features that the engines support, typescript won't complain if you use them.